src/stp/new_plays/AttackingPass.cpp

```diff
@@ -53,12 +60,14 @@ Dealer::FlagMap AttackingPass::decideRoleFlags() const noexcept {
     flagMap.insert({"test_role_7", {not_important}});
     flagMap.insert({"test_role_8", {not_important}});
     flagMap.insert({"test_role_9", {not_important}});
-    flagMap.insert({"test_role_10", {not_important}});
 
     return flagMap;
 }
 
 void AttackingPass::calculateInfoForRoles() noexcept {
+    stpInfos["keeper"].setEnemyRobot(world->getWorld()->getRobotClosestToBall(world_new::them));
+    stpInfos["keeper"].setPositionToShootAt(Vector2());
```

Comment:
Huh? You shoot to an undefined position?

Reply:
Nope, this is a default constructed Vector2, which is (0,0). This is temporary, but changing this to a position that makes more sense is not for this PR but a future one.